mtp_cashbook/apps/disbursements/views.py

```diff
@@ -283,7 +289,9 @@ def get_context_data(self, **kwargs):
         context['remittance_description_attrs'] = {
             'maxlength': field.max_length,
         }
-        return context
+
+        kwargs.update(self.get_valid_form_data(PrisonerView))
```

Comment:
these 2 lines appear to entirely replace the context that's created in this method above. this isn't the intention